.gitignore

```diff
@@ -88,3 +88,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+# For Kushi's Postgres installation
+kushi_db.sh
```

Comment:
You can also add the file to .git/info/exclude if it's just a local file.

See the last section here:
https://help.github.com/articles/ignoring-files/